examples/nativemodules/PureNativeModule.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { fireEvent, render } from 'react-native-testing-library';
+import PureNativeModule from './PureNativeModule';
+
+jest.mock('react-native', () => {
```

Comment:
Also, I think we should be able to mock like this:
```js
import { NativeModules, UIManager } from 'react-native';

jest.spyOn(NativeModules, 'NativeNotification').mockImplementation(() => ({show: jest.fn()}));
jest.spyOn(UIManager, 'RNCSafeAreaView').mockImplementation(() => ({}));
```

I think that would be cleaner than using `jest-mock`. The downside is that `jest.spyOn` is not hoisted, so if a module has side-effects and calls NativeModules/UIManager at import time, it probably won't work